eslint.config.js

```diff
@@ -0,0 +1,310 @@
+// @ts-check
+const emotionPlugin = require('@emotion/eslint-plugin');
+const { fixupPluginRules } = require('@eslint/compat');
```

Comment:
Actually, 40 minutes ago at the time of writing, a maintainer said that a PR that will help add compatibility for the eslint plugin will be reviewed/merged soon! 
So might be worth us checking the status of [this PR](https://github.com/testing-library/eslint-plugin-testing-library/pull/925#issuecomment-2418896886) before we merge 🙌

Reply:
@tomratcliffe the PR was [merged 10hrs ago](https://github.com/testing-library/eslint-plugin-testing-library/pull/925#issuecomment-2420735710). I'm happy for us to get this out the door first and then bump it and clean up.